concrete/src/Permission/Access/Entity/PageOwnerEntity.php

```diff
@@ -29,23 +29,21 @@ public function getAccessEntityUsers(PermissionAccess $pae)
         }
         if (is_object($c) && ($c instanceof Page)) {
             $ui = UserInfo::getByID($c->getCollectionUserID());
-            $users = array($ui);
-
-            return $users;
+            if ($ui) {
+                return [$ui];
```

Comment:
This changes behavior a little bit, should we be sure to return an empty array if !$ui?